scripts/make_changelog.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+COMMITS=$(git log --merges --pretty=format:"%H" origin/master)
+
+GIT_USER_NAME=$(git config user.name)
+GIT_USER_EMAIL=$(git config user.email)
+DATE=$(date '+%Y-%m-%d')
+
+MARKDOWN="## Changelog \n ### Generated by [$GIT_USER_NAME]($GIT_USER_EMAIL) on ${DATE}"
```

Comment:
Not sure we need to show `Generated by [$GIT_USER_NAME]($GIT_USER_EMAIL)`

Reply:
Sure, I will remove.